lib/merge.js

```diff
@@ -56,6 +68,29 @@ async function merge(context, pullRequest) {
   return true;
 }
 
+async function removeLabels(octokit, pullRequest, mergeRemoveLabels) {
+  const pullRequestLabels = pullRequest.labels.map(label => label.name);
```

Comment:
I think you can inline pullRequestLabels

Reply:
Fixed.